iceoryx_hoofs/primitives/include/iox/iceoryx_hoofs_types.hpp

```diff
@@ -24,7 +24,10 @@
 
 namespace iox
 {
-using byte_t = uint8_t;
+
+enum class byte : uint8_t
```

Comment:
Just one little thingy. In our Coding Guidelines we state that every type has to be written in `UpperCamelCase`. We should adjust `byte` into `Byte`.

Reply:
Well, in general you are right but for STL replacements we use or at least used to use lower_snake_case

Reply:
Oh I see ... I wasn't aware that this is also in C++17. Then lets leave it as it is.

Reply:
@elfenpiff is not `byte` here a porting for `std::byte` and therefore we use std naming convention and start the type with small character ?

Reply:
@elfenpiff forget my comment, did not refresh the page to see @elBoberido 's :laughing: